### PR TITLE
Update office-365-multi-geo.md

### PR DESCRIPTION
**Pull request by msbemba**
Made corrections to reflect the minimum number of licenses required and added link to the Microsoft blog for more details

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/811 and #https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/812

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/office-365-multi-geo.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Follow these steps to get started with multi-geo:
 
 5. [Set up search](configure-search-for-multi-geo.md).
 
+> [!NOTE]
+>For more details on Licensing and Pricing visit [Multi Geo licensing and Pricing](https://techcommunity.microsoft.com/t5/blogs/blogarticleprintpage/blog-id/Office365Blog/article-id/2176)
```
Comment (JohanFreelancer9) [inline Enterprise/office-365-multi-geo.md:71]:
```suggestion
> For more details on licensing and pricing, visit [Multi-geo licensing and pricing](https://techcommunity.microsoft.com/t5/blogs/blogarticleprintpage/blog-id/Office365Blog/article-id/2176).
```

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue  #811 and #812.

Thank you.

cc: @kenwith

**Comment by MikePlumleyMSFT** [comment]
The blog post is a point in time snapshot. We would not link to that here as a source of up-to-date information.

**Pull request merged**